example2/ts-frontend/src/greeter.ts

```diff
@@ -0,0 +1,15 @@
+interface Person {
```

Comment:
@saberduck do you have any issue in this code?

Reply:
Added violation of https://rules.sonarsource.com/typescript/RSPEC-3786